File: src/app/personas/persona.ts
export interface Persona {
    id?: string | null | undefined;
    nombreCompleto?: string | null | undefined;
    identificacion?: string | null | undefined;
    edad?: number | null | undefined;
    genero?: string | null | undefined;
    estaActiva?: boolean | null | undefined;
    atributosAdicionales?: string | null | undefined;
    maneja?: boolean | null | undefined;
    usaLentes?: boolean | null | undefined;
    diabetico?: boolean | null | undefined;
    padeceOtraEnfermedad?: boolean | null | undefined;
    cualOtraEnfermedad?: string | null | undefined;
  }

File: Seguros.Backend/Seguros.Domain/Entities/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Seguros.Domain.Entities
{
    public class Persona
    {
        public Guid Id { get; set; }
        public string NombreCompleto { get; set; }
        public string Identificacion { get; set; }
        public int Edad {  get; set; }
        public string Genero { get; set; }
        public bool EstaActiva { get; set; }
        public string? AtributosAdicionales { get; set; } = null;
        public bool Maneja { get; set; }
        public bool UsaLentes { get; set; }
        public bool Diabetico { get; set; }
        public bool PadeceOtraEnfermedad { get; set; }
        public string? CualOtraEnfermedad { get; set; }

    }
}
